assets/js/base/context/cart-checkout/payment-methods/payment-method-data-context.js

```diff
@@ -337,23 +337,35 @@ export const PaymentMethodDataProvider = ( { children } ) => {
 	// Set active (selected) payment method as needed.
 	useEffect( () => {
 		const paymentMethodKeys = Object.keys( paymentData.paymentMethods );
+		const allPaymentMethodKeys = [
+			...paymentMethodKeys,
+			...Object.keys( paymentData.expressPaymentMethods ),
+		];
```

Comment:
Not really. The first condition still only cares about paymentMethodKeys because we only _update_ `activePaymentMethod` _from_ paymentMethods (not express payment method) when it's empty. So the first condition bailing early when there are no registered payment methods is still valid.